### PR TITLE
Footer Added

### PR DESCRIPTION
**Pull request by Rajdip019**
#17 
Here is the footer

Screenshot 2022-03-02 121144

### CODE REVIEW

File: client/pages/index.js [hunk 1]
```diff
@@ -27,36 +27,41 @@ export default function Home() {
    * @returns {Promise<void>}
    */
   async function loadNFTs() {
-    
     // For the Mumbai Testnet
     // const provider = new ethers.providers.JsonRpcProvider(rpc_url);
 
     // For the LocalHost
-    // const provider = new ethers.providers.JsonRpcProvider();
+    const provider = new ethers.providers.JsonRpcProvider();
```
Comment (Megabyte-143) [inline client/pages/index.js:34]: Please comment this line(**34**).

**Comment by raysummee** [comment]
@Rajdip019 can you only add the file which you have made changes. Not all. Only add the Footer.js file. 

I can see the files like index.js and contracts too.

**Comment by Rajdip019** [comment]
Thanks a lot.

**Review by raysummee — CHANGES_REQUESTED** [review]
@Radip019 Please add the Footer into _app.js not in index.js. By using _app.js, we don't have to include the footer for each page. After that I will merge this.

**Comment by Rajdip019** [comment]
I have  made the changes. And another Question I have found some Issues and want on work on that and also want to work in here for 3 months long, building the whole UI of the website. So, should I just make new Issues explaining them and after you assign me I can make the changes and make a PR right? @raysummee

File: client/pages/index.js (hunk 1 above)
Comment (raysummee) [reply client/pages/index.js:33]: @Rajdip019 I might be wrong, but you might have forget to comment it after the changes. line(33). Refer to https://github.com/Megabyte-143/certified_cliche/pull/33/files#r817398784

**Review by raysummee — APPROVED** [review]

**Comment by raysummee** [comment]
@Rajdip019  Sure. I will assign you another issue after this one then. Thank you for the contribution.

**Comment by raysummee** [comment]
@Rajdip019 you have requested PR to main. Should be dev. Let me revert it after that I'll merge it into dev

**Pull request merged**